packages/smooth_app/lib/pages/scan_page.dart

```diff
@@ -1,25 +1,34 @@
 import 'package:flutter/material.dart';
+
 import 'package:provider/provider.dart';
 import 'package:smooth_app/data_models/continuous_scan_model.dart';
 import 'package:smooth_app/database/local_database.dart';
-import 'package:smooth_app/pages/alternative_continuous_scan_page.dart';
-import 'package:smooth_app/pages/continuous_scan_page.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:smooth_ui_library/widgets/smooth_toggle.dart';
 import 'package:smooth_app/database/product_query.dart';
+import 'package:smooth_app/lists/smooth_product_carousel.dart';
+import 'package:smooth_app/pages/personalized_ranking_page.dart';
+import 'package:smooth_ui_library/animations/smooth_reveal_animation.dart';
+import 'package:smooth_ui_library/widgets/smooth_view_finder.dart';
+import 'package:qr_code_scanner/qr_code_scanner.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 class ScanPage extends StatelessWidget {
   const ScanPage({
     @required this.contributionMode,
     @required this.mlKit,
+    this.continuousScanModel,
```

Comment:
It make no sense to use `continuousScanModel` as a class parameter and as a `Future` variable.